HW0/P4/P4.py

```diff
@@ -40,37 +42,132 @@
     # Read the observation array and plot it (Part 2)
     #####################
 
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '.g', label='Observed trajectory')
+    C_partial = np.matrix([
```

Comment:
It's generally recommended to use NumPy ndarrays rather than matrices, although I think they said to use matrices in class so it should be fine.